tests/core/editable/keystrokes/delbackspacequirks/manual/nbspdeleting.html

```diff
@@ -0,0 +1,11 @@
+<textarea id="editor" cols="30" rows="10">
+	<p>Hello World!</p>
+</textarea>
+
+<script>
+	if ( !CKEDITOR.env.webkit ) {
```

Comment:
Also here - use `CKEDITOR.env.chrome`. And this test should be ignored on mobile devices as they usually don't have `delete` key.

Reply:
See https://github.com/ckeditor/ckeditor4/pull/4746#discussion_r650813470 - so only add ignoring on mobile.